docs/options.rst

```diff
@@ -284,6 +284,14 @@ Boolean. Default: false
 If true, selecting a year or month in the datepicker will update the input value immediately. Otherwise, only selecting a day of the month will update the input value immediately.
 
 
+keepEmptyValues
+---------------
+
+Boolean. Default: false
```

Comment:
Can you add the option an its default value to the overview at the bottom of this docs file